email-templates/src/templates/grant-waiting-list.html

```diff
@@ -0,0 +1,93 @@
+                <table class="w-full">
+                  <tr>
+                    <td
+                      class="bg-white text-gray-800 text-base text-left p-48 sm:px-24 leading-24"
+                    >
+                      <p class="m-0 mb-16">👋 @{{firstname}}!</p>
+                      <p class="m-0 mb-16">
+                        Thank you for submitting a grant request for @{{conferenceName}}.
+                      </p>
+                      <p class="m-0 mb-16">
+                        The number of requests to the PyCon Italia Financial Aid
+                        program was overwhelming this year, and we received more
+                        applications than we can accommodate in the program.
+                      </p>
+                      <p class="m-0 mb-16">
+                        Your request is currently in our <strong>waiting list</strong>,
+                        if more funds become available or if a spot becomes available,
+                        we will inform you by @{{grantsUpdateDeadline}}
+                      </p>
+                      <p class="m-0 mb-24">
+                        If for some reason you no longer need the grant, or you
+                        have any questions or need more information please let
+                        us know by clicking the link below.
+                      </p>
+
+                      <div
+                        class="leading-full items-center justify-center flex mb-24"
+                      >
+                        <a
+                          href="@{{replyLink}}"
+                          target="_blank"
+                          class="inline-block py-16 px-24 select-none text-base font-semibold text-center no-underline text-white bg-primary hover:bg-darker"
+                        >
+                          <!--[if mso
+                            ]><i
+                              style="
+                                letter-spacing: 24px;
+                                mso-font-width: -100%;
+                                mso-text-raise: 30px;
+                              "
+                              >&#8202;</i
+                            ><!
+                          [endif]-->
+                          <span style="mso-text-raise: 16px"
+                            >Manage your Grant</span
+                          >
+                          <!--[if mso
+                            ]><i
+                              style="
+                                letter-spacing: 24px;
+                                mso-font-width: -100%;
+                              "
+                              >&#8202;</i
+                            ><!
+                          [endif]-->
+                        </a>
+                      </div>
+                      <p class="m-0 mb-24">
+                        Thank you very much for your enthusiasm and interest in
+                        PyCon Italia.
```

Comment:
nitpick: fine for now, but if we use a variable conference name we should it everywhere

Reply:
changed with the conference